bigbluebutton-html5/imports/api/audio/client/bridge/sip.js

```diff
@@ -324,7 +325,21 @@ class SIPSession {
       if (browser().name === 'edge') {
         connectionCompletedEvents = ['iceConnectionCompleted'];
       }
-
+      //getStats() API
+      const getWebRTCStats = (peer) => { 
+        (async () => {
+          const report = await peer.getStats();
+          let allRtcStats = new Array();
+          for (let item of report.values())                          
+            allRtcStats.push(item);
+          let result = {};                           
+          result['enabled']= this.webrtcConnected;
+          result['timer']=  Date.now();
+          result['rtcStats']= allRtcStats;
+          logger.info({ logCode: 'webRTCStats' }, JSON.stringify(result));                       
```

Comment:
We've moved away from logging JSON stringified data. The data should be under `extraInfo` like here, https://github.com/bigbluebutton/bigbluebutton/commit/1b94039a4f50f556f487e5861e615cab77c8d4da, and the message should be a human readable string that clearly shows what it is.

The logCode should also be unique and would be better as `sip_js_webrtc_stats`